modulefiles/gsi_hera.intel.lua

```diff
@@ -1,12 +1,12 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev-rocky8/install/modulefiles/Core")
+prepend_path("MODULEPATH", '/scratch1/NCEPDEV/global/David.Huber/SPACK/ss_1.8.0/envs/gsi-addon/install/modulefiles/Core')
```

Comment:
@DavidHuber-NOAA , when will the official `spack-stack/1.8` be released on Hera and other NOAA RDHPCS.  What's the plan for WCOSS2?

Reply:
Thank you @DavidHuber-NOAA.  The disconnect between the NOAA RDHPCS library system and WCOSS2 is troubling.  Operations runs on WCOSS2.  Much of our development and parallel testing occurs on NOAA RDHPCS. This opens the door to potential issues when we transition development to operations.  I understand that you can not do anything about this.  Management is aware of this. It would be good to remind them that this remains an issue.

Reply:
@RussTreadon-NOAA I apologize, BUFR will also be upgraded to 12.1.0 on WCOSS2.  That is a requirement for this PR.  That said, other libraries will likely still lag behind (e.g. netCDF, HDF5).

Reply:
Thank you @DavidHuber-NOAA for your diligence in keeping various applications up to date with library releases.